.github/workflows/ping-other-repos.yml

```diff
@@ -0,0 +1,25 @@
+name: Remote Dispatch Action Initiator
+
+on:
+  push:
```

Comment:
Do you want to be pinged on every push to every branch on mdn/browser-compat-data? Or just to `main`?

Reply:
> Do you want to be pinged on every push to every branch on mdn/browser-compat-data? Or just to `main`?

Good point — there’s no need for this to run on pushes to branches, so I changed to run only for pushes to the main branch